src/transport.cpp

```diff
@@ -628,6 +628,7 @@ namespace quicr {
     void Transport::SubscribeTrack(TransportConnId conn_id, std::shared_ptr<SubscribeTrackHandler> track_handler)
     {
         const auto& tfn = track_handler->GetFullTrackName();
+        track_handler->connection_handle_ = conn_id;
```

Comment:
This was missing, which caused the track handler to have a connection handle/ID of zero, which broke cleanup on subscribe error.  Super simple fix. :)